tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -316,3 +316,71 @@ if (!/\/2\.7/.test(Cypress.env('rancher_version'))) {
     )  
   });
 
+  describe('Test Self-Healing of IMMUTABLE resources modification when correctDrift option used for exisiting GitRepo', { tags: '@p1'}, () => {
+    const correctDriftTestData: testData[] = [
+      { qase_id: 80,
+        repoName: "local-cluster-correct-80",
+        resourceType: "ConfigMaps",
+        resourceName: "mp-app-config",
+        resourceNamespace: "test-fleet-mp-config",
+      },
+      { qase_id: 79,
+        repoName: "local-cluster-correct-79",
+        resourceType: "Services",
+        resourceName: "mp-app-service",
+        resourceNamespace: "test-fleet-mp-service"
+      },
+    ]
+
+    correctDriftTestData.forEach(
+      ({qase_id, repoName, resourceType, resourceName, resourceNamespace}) => {
```

Comment:
Add `dataToAssert` param if accept the previous option
```suggestion
      ({qase_id, repoName, resourceType, resourceName, resourceNamespace, dataToAssert}) => {
```

Reply:
Done